### PR TITLE
Style tweaks for things that got borked in the merges

### PR DESCRIPTION
**Pull request by Camwyn**
Some things got lost in the shuffle - this replaces/corrects them.

Ticket: https://central.tri.be/issues/80630

~Linked PR: https://github.com/moderntribe/event-tickets-plus/pull/352~ (merged)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/2473026b93542b3781a518fe9e87bbdd4f283a5c/src/admin-views/edit_admin_panel.php#L104)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/edit_admin_panel.php#L104)
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [list.php:60](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/list.php#L60)
:x: | There must be a single space after the first parenthesis | [list.php:60](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/list.php#L60)
:x: | There must be a single space before the last parenthesis | [list.php:60](https://github.com/moderntribe/event-tickets/blob/d7c14c5de7a7db10c77f3a431e2ffd753471f07d/src/admin-views/list.php#L60)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/d09ba235158b79bd7049baa97276bb2398acb712/src/admin-views/edit_admin_panel.php#L104)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Ticket_Object.php [hunk 1]
```diff
@@ -407,6 +407,26 @@ public function display_original_stock() {
 			return $orginal_stock;
 		}
 
+
+		/**
+		 * Method to display the available `stock` propriety of the Object.
```
Comment (zbtirrell) [inline src/Tribe/Ticket_Object.php]: typo - "propriety" should be "property"

File: src/Tribe/Ticket_Object.php [hunk 2]
```diff
@@ -407,6 +407,26 @@ public function display_original_stock() {
...
+		 *
+		 * Returns the current ticket available capacity: either an integer or a
+		 * string if stock is unlimited.
+		 *
+		 * @return int|string
+		 */
+		public function display_available_stock() {
```
Comment (zbtirrell) [inline src/Tribe/Ticket_Object.php]: I find this method name weird. It does not actually "display" anything.

File: src/admin-views/edit_admin_panel.php [hunk 3]
```diff
@@ -16,16 +16,17 @@
...
 				<div class="input_block">
-					<label class="ticket_form_label" for="ticket_name"><?php esc_html_e( 'Type:', 'event-tickets' ); ?></label>
-					<input type='text' id='ticket_name' name='ticket_name' class="ticket_field" size='25' value='' />
+					<label class="ticket_form_label ticket_form_left" for="ticket_name"><?php esc_html_e( 'Type:', 'event-tickets' ); ?></label>
+					<input type='text' id='ticket_name' name='ticket_name' class="ticket_field ticket_form_right" size='25' value='' />
+					<span class="tribe_soft_note ticket_form_right"><?php esc_html_e( 'Ticket type name shows on the front end and emailed tickets', 'event-tickets' ); ?></span>
 				</div>
 				<fieldset class="input_block">
-					<legend class="ticket_form_label"><?php esc_html_e( 'Sell using:', 'event-tickets' ); ?></legend>
+					<legend class="ticket_form_label"><?php esc_html_e( 'Sell using (visible for testing):', 'event-tickets' ); ?></legend>
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: "(visible for testing)" <= was this left in by accident?

File: src/admin-views/edit_admin_panel.php [hunk 4]
```diff
@@ -16,16 +16,17 @@
...
 					<?php
 					$checked = true;
 					foreach ( $modules as $class => $module ) {
 						?>
-						<input <?php checked( $checked ); ?> type="radio" name="ticket_provider" id="<?php echo esc_attr( $class . '_radio' ); ?>" value="<?php echo esc_attr( $class ); ?>" class="ticket_field ticket_provider" tabindex="-1">
+						<input <?php checked( $checked ); ?> type="radio" name="ticket_provider" id="<?php echo esc_attr( $class . '_radio' ); ?>" value="<?php echo esc_attr( $class ); ?>" class="ticket_field ticket_provider " tabindex="-1">
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: stray space in class attribute

File: src/admin-views/edit_admin_panel.php [hunk 5]
```diff
@@ -59,50 +60,53 @@
 				<section id="ticket_form_advanced" class="advanced accordion-content">
 					<h4 class="accordion-label"><?php esc_html_e( 'Advanced Settings', 'event-tickets' ); ?></h4>
 					<div class="input_block">
-						<label class="ticket_form_label" for="ticket_description"><?php esc_html_e( 'Ticket Description:', 'event-tickets' ); ?></label>
-						<textarea rows="5" cols="40" name="ticket_description" class="ticket_field" id="ticket_description"></textarea>
+						<label class="ticket_form_label ticket_form_left" for="ticket_description" ><?php esc_html_e( 'Ticket Description:', 'event-tickets' ); ?></label>
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: stray space in `label` tag

File: src/admin-views/edit_admin_panel.php [hunk 6]
```diff
@@ -59,50 +60,53 @@
...
-							</select>
-							<select name="ticket_start_minute" id="ticket_start_minute" class="ticket_field tribe-dropdown">
-								<?php echo $startMinuteOptions; ?>
-							</select>
-							<?php if ( ! strstr( get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT ), 'H' ) ) : ?>
-								<select name="ticket_start_meridian" id="ticket_start_meridian" class="ticket_field tribe-dropdown">
-									<?php echo $startMeridianOptions; ?>
+						<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+						<div class="ticket_form_right">
+							<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_start_date" id="ticket_start_date" value="" >
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: empty `value` attribute is not needed

File: src/admin-views/edit_admin_panel.php [hunk 7]
```diff
@@ -59,50 +60,53 @@
...
+								<select name="ticket_start_hour" id="ticket_start_hour" class="ticket_field tribe-dropdown">
+									<?php echo $startHourOptions; ?>
 								</select>
-							<?php endif; ?>
-						</span>
+								<select name="ticket_start_minute" id="ticket_start_minute" class="ticket_field tribe-dropdown">
+									<?php echo $startMinuteOptions; ?>
+								</select>
+								<?php if ( ! strstr( get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT ), 'H' ) ) : ?>
+									<select name="ticket_start_meridian" id="ticket_start_meridian" class="ticket_field tribe-dropdown">
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: this is a long standing issue, and probably not worth fixing, but: it should be "meridiem" not "meridian"

File: src/admin-views/edit_admin_panel.php [hunk 8]
```diff
@@ -59,50 +60,53 @@
...
+								<?php endif; ?>
+							</span>
+						</div>
 					</div>
 					<div class="input_block">
-						<label class="ticket_form_label" for="ticket_end_date"><?php esc_html_e( 'End sale:', 'event-tickets' ); ?></label>
-						<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_end_date" id="ticket_end_date" value="">
+						<label class="ticket_form_label ticket_form_left" for="ticket_end_date"><?php esc_html_e( 'End sale:', 'event-tickets' ); ?></label>
+						<div class="ticket_form_right">
+							<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_end_date" id="ticket_end_date" value="">
```
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: blank value attribute should be removed

File: src/resources/postcss/tickets-refresh.pcss [hunk 9]
```diff
@@ -210,6 +255,27 @@
 	}
 }
 
+/* Adjust font size for entire form */
+#event_tickets {
+	font-size: 12px;
```
Comment (zbtirrell) [inline src/resources/postcss/tickets-refresh.pcss]: seems odd to be using a mix of `em` and `px`, but maybe it is for good reason ¯\_(ツ)_/¯

File: src/admin-views/edit_admin_panel.php (hunk 3 above)
Comment (Camwyn) [inline src/admin-views/edit_admin_panel.php]: No - that's there until I get the capacity done (hidden in that branch), but it's not supposed to show up, so I left a note for Leah or whoever tests it so they know I'm aware and it's being removed.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:74](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startHourOptions' | [edit_admin_panel.php:76](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMinuteOptions' | [edit_admin_panel.php:79](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L79)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$startMeridianOptions' | [edit_admin_panel.php:83](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [edit_admin_panel.php:95](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L95)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endHourOptions' | [edit_admin_panel.php:97](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L97)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMinuteOptions' | [edit_admin_panel.php:100](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L100)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$endMeridianOptions' | [edit_admin_panel.php:104](https://github.com/moderntribe/event-tickets/blob/41ba7ad3ae47ec3e4b86e5aea2aa6ca1cb093194/src/admin-views/edit_admin_panel.php#L104)

> via jenkins/codesniffer-phpcs

File: src/admin-views/edit_admin_panel.php (hunk 7 above)
Comment (Camwyn) [inline src/admin-views/edit_admin_panel.php]: Yeah, let's do that in a separate ticket/PR so we can be sure we're not changing something that isn't in the database and doesn't break compatibility.

File: src/resources/postcss/tickets-refresh.pcss (hunk 9 above)
Comment (Camwyn) [inline src/resources/postcss/tickets-refresh.pcss]: Setting a base font size for the component (in px - could use rem?) that everything inside scales from.

File: src/admin-views/edit_admin_panel.php (hunk 3 above)
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: 👍

File: src/admin-views/edit_admin_panel.php (hunk 7 above)
Comment (zbtirrell) [inline src/admin-views/edit_admin_panel.php]: 💯

File: src/resources/postcss/tickets-refresh.pcss (hunk 9 above)
Comment (zbtirrell) [inline src/resources/postcss/tickets-refresh.pcss]: 👍

**Pull request merged**